src/System.Runtime.InteropServices.RuntimeInformation/src/RuntimeInformation.cs

```diff
@@ -7,7 +7,13 @@ namespace System.Runtime.InteropServices
 {
     public static partial class RuntimeInformation
     {
+#if netcore50aot
+        private const string FrameworkName = ".NET Native";
+#elif net46
```

Comment:
Currently we only have the targeting pack for net46. We should decide how far downlevel we need to support this and then target and we can add the supporting targeting pack for those.